shared/actions/login/index.js

```diff
@@ -392,7 +392,16 @@ const makeKex2IncomingMap = (dispatch, getState, onBack: SimpleCB, onProvisioner
     'keybase.1.provisionUi.DisplayAndPromptSecret': ({phrase, secret}, response) => {
       dispatch({payload: {textCode: new HiddenString(phrase)}, type: Constants.setTextCode})
       generateQRCode(dispatch, getState)
-      dispatch(askForCodePage(phrase => { response.result({phrase, secret: null}) }, () => onBack(response)))
+      let codeSent = false
```

Comment:
I know this is a quick fix on top of some really really old code but we should avoid this kind of pattern of passing callbacks through our actions and store. Saga-izing this would really help clean this stuff up